apps/arena/lib/arena_web/controllers/health_controller.ex

```diff
@@ -8,8 +8,11 @@ defmodule ArenaWeb.HealthController do
   end
 
   def version(conn, _params) do
+    arena_version = Application.spec(:arena, :vsn) |> to_string()
+    configurator_version = GameBackend.Configuration.get_configuration_hash_version()
```

Comment:
This is not correct

Reply:
Be careful, this controller is duplicated in both arena and gateway apps.
We should distinct them somehow or see if both are needed.